build/mvn

```diff
@@ -78,7 +78,7 @@ install_app() {
       echo "  ${local_tarball}" >> ${local_checksum} # two spaces + file are important!
       # Assuming SHA512 here for now
       echo "Veryfing checksum from ${local_checksum}" 1>&2
-      if ! shasum -a 512 -q -c "${local_checksum}" ; then
+      if ! shasum -a 512 -c "${local_checksum}" > /dev/null 2>&1 ; then
```

Comment:
Shall we remove "2>&1"? So that the error message can be shown

Reply:
Make sense, just let wrong message print.

Reply:
please update the PR description, too.

Reply:
Done